src/Microsoft.CodeAnalysis.Razor.Workspaces/TagHelperFactsServiceInternal.cs

```diff
@@ -1,20 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.CodeAnalysis.Host;
-
-namespace Microsoft.CodeAnalysis.Razor
-{
-    internal abstract class TagHelperFactsServiceInternal : ILanguageService
```

Comment:
I'm wonder if we will eventually pivot back the other way on where the implementation of these types live, but I'm going to trust you on this for now.